src/templates/en/2019/chapters/css.html

```diff
@@ -0,0 +1,19 @@
+
+{% extends "en/base.html" %}
+
+{% block title %}
+CSS
+{% endblock %}
+
+{% block meta %}
+  <meta name="description" content="Something about CSS">
+{% endblock %}
+
+{% block main %}
+<h1>Chapter 2: CSS</h1>
+<p>This is a test.</p>
+<p>And <strong>only</strong> a test</p>
+<blockquote><p>If it was anything more, it would certainly have had better placeholder text</p>
+</blockquote>
+
```

Comment:
The trailing newline is because of the template. I can put the block on a single line, but I feel that it loses some readability. Would you like me to change it?

```
{{% block main %}}
{body}
{{% endblock %}}
```

Reply:
In that case don't worry about it.